kind-up.sh

```diff
@@ -0,0 +1,83 @@
+#!/bin/sh
```

Comment:
If we are going to add this utility we should get this somewhere else (might be under scripts or under test) and document it. Given that this is not core of the goal of this PR I suggest to remove it from this changes set and eventually to address it in a separated PR

Reply:
I'll plan to remove this from this PR...